src/functionalTest/java/uk/gov/hmcts/reform/blobrouter/BulkScanBlobDispatchingTest.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.blobrouter;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.google.common.io.Resources.getResource;
+import static com.google.common.io.Resources.toByteArray;
+import static com.jayway.awaitility.Awaitility.await;
+import static java.util.Arrays.asList;
+import static org.hamcrest.Matchers.equalTo;
+import static org.springframework.http.HttpStatus.OK;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Status.DISPATCHED;
+import static uk.gov.hmcts.reform.blobrouter.envelope.ZipFileHelper.createZipArchive;
+import static uk.gov.hmcts.reform.blobrouter.storage.StorageHelper.blobExists;
+import static uk.gov.hmcts.reform.blobrouter.storage.StorageHelper.uploadFile;
+
+public class BulkScanBlobDispatchingTest extends FunctionalTestBase {
+
+    private static final String CONTAINER = "bulkscan";
+
+    @Test
+    void should_move_blob_to_bulkscan_storage() throws Exception {
+        // upload bulkscan file with unique name
+        String fileName = randomFileName();
+
+        byte[] internalZipContent = toByteArray(
+            getResource("test-data/envelope/envelope.zip")
+        );
+
+        byte[] wrappingZipContent = createZipArchive(
+            asList("test-data/envelope/envelope.zip", "test-data/envelope/signature")
+        );
```

Comment:
This is fine, although for CFT we can reduce these two statements to just one. For crime it matters that there's a zip containing another zip, because blob-router is supposed to extract it and only send the internal one. For CFT blobs content doesn't matter much (although in the future it will, to some extent). Up to you.

Reply:
non-repudiation will be applied to all blobs disregarding the source container. so even content doesn't matter, but signature does

Reply:
fair enough